### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by binyang97** [yyliu01/IT2#5]
Hello, 

there are only pretrained weights with 10% label regime provided in the repository. As I'm using different GPUs and batch size for training, the performance is slightly different from what you presented in the paper. As I want to test some post-training method, it would be very helpful if you could share me  the pretrained weights of other configurations. Thanks.

Best, 
Bin Yang

**Comment by yyliu01** [comment]
Hey @binyang97 

I apologise for the late reply. Due to my relocation from Australia to the UK, most of the checkpoints for the SemanticKITTI dataset were lost. At the moment, after a thorough search, I’ve only been able to locate the NuScenes checkpoints in my Google Drive. 

1pct: https://drive.google.com/drive/folders/1kROsfPzlus0hJuBxi-G7U1p0S9S_mCt-?usp=drive_link
10pct: https://drive.google.com/drive/folders/1E9-MjNhTq6ToLLoeTuePRkeOGk5uzyEo?usp=drive_link
20pct: https://drive.google.com/drive/folders/14uoSzTFtWzufqoDFZI6QpSr-GTk5URrv?usp=drive_link
50pct: https://drive.google.com/drive/folders/11jzQb4KplIDCoEQeiJeizTXUyI2XLFzg?usp=drive_link

Please note that some of these results are based on reimplementations and may be slightly higher than those reported in the original paper.

Hope it helps.

Cheers,
Yuyuan

**Comment by binyang97** [comment]
> Hey [@binyang97](https://github.com/binyang97)
> 
> I apologise for the late reply. Due to my relocation from Australia to the UK, most of the checkpoints for the SemanticKITTI dataset were lost. At the moment, after a thorough search, I’ve only been able to locate the NuScenes checkpoints in my Google Drive.
> 
> 1pct: https://drive.google.com/drive/folders/1kROsfPzlus0hJuBxi-G7U1p0S9S_mCt-?usp=drive_link 10pct: https://drive.google.com/drive/folders/1E9-MjNhTq6ToLLoeTuePRkeOGk5uzyEo?usp=drive_link 20pct: https://drive.google.com/drive/folders/14uoSzTFtWzufqoDFZI6QpSr-GTk5URrv?usp=drive_link 50pct: https://drive.google.com/drive/folders/11jzQb4KplIDCoEQeiJeizTXUyI2XLFzg?usp=drive_link
> 
> Please note that some of these results are based on reimplementations and may be slightly higher than those reported in the original paper.
> 
> Hope it helps.
> 
> Cheers, Yuyuan

Thank you so much!
